notebooks/quality_control.ipynb

```diff
@@ -0,0 +1,103 @@
+    "\n",
+    "In quality control for cell detection, the mask area is used as a metric to evaluate the accuracy and consistency of the segmentation process. Specifically, the mask area can be compared to the expected area of a single cell, which may be determined from prior knowledge or reference data. If the mask area is significantly different from the expected area, it may indicate errors or inconsistencies in the segmentation process.\n",
+    "\n",
+    "For example, if the mask area is much larger than expected for a single cell, it may indicate that the segmentation process is including pixels that do not belong to the cell, leading to false positives. This can result from a variety of factors, including variations in staining, illumination, or noise. On the other hand, if the mask area is much smaller than expected, it may indicate that the segmentation process is missing pixels that belong to the cell, leading to false negatives.\n",
+    "\n",
+    "By monitoring the mask area as a quality control metric, image segmentation processes for cell detection can be improved to ensure that they are accurate and consistent. This can involve adjusting parameters, such as threshold values or filter settings, or using more advanced segmentation techniques, such as deep learning approaches.\n",
+    "\n",
+    "**roundness:** Roundness is another quality control metric used to evaluate the shape of segmented cells (Tegtmeier et al.,2018). Roundness measures how closely a cell's shape resembles a perfect circle. It is calculated as the ratio of the cell's actual area to the area of a circle with the same perimeter. A perfectly round cell will have a roundness value of 1, while more elongated or irregular cells will have lower roundness values.\n",
+    "\n",
+    "Roundness is a useful metric in cell detection because it can help identify cells that have been improperly segmented. For example, cells that are elongated or irregular in shape may have been segmented incorrectly due to noise, overlapping cells, or other factors. By comparing the roundness of segmented cells to a threshold value, cells that are improperly segmented can be identified and corrected.\n",
+    "\n",
+    "### 2.2 Trace\n",
+    "**skewness (Stringer & Pachitariu, 2019):** The skewness of fluorescence activity from multi-day long experiments can be used as a quality control metric by providing an indication of the consistency and stability of the signal over time. A stable signal with consistent skewness values across multiple days suggests that the measurement system is working properly and the experimental conditions are consistent. However, changes in the skewness of fluorescence activity over time could indicate technical issues or changes in the sample physiology or behavior, which could affect the quality of the data and the accuracy of any conclusions drawn from the experiment.\n",
```

Comment:
Please link the paper. This citation is not sufficient to find the paper.

Reply:
I'll add the links for all papers.

Reply:
Actually in this case, we calculate the skewness for each frame. I need to change the explanation accordingly.